backend/src/v1/complaint/complaint.service.ts

```diff
@@ -801,6 +801,7 @@ export class ComplaintService {
       builder.where("complaint.complaint_identifier = :id", { id });
       const result = await builder.getOne();
 
+      const test = 0;
```

Comment:
Not sure why Sonar didn't flag this, but this should not be here.

Reply:
I am equally surprised by that as well